xml/net_samba.xml

```diff
@@ -778,13 +730,74 @@
     configuration.
    </para>
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
+   </para>
+   <note>
+    <title>SMB1 Deprecation Only Affects Clients Using <command>mount</command></title>
+    <para>
+     SMB1 is still supported by the Samba server software shipped with
+     <!-- NB: profiling below is intentional - we don't support using SLED as a
+     Samba server. -->
+     <phrase os="sles;sled">&sls;</phrase><phrase os="osuse">&osuse;</phrase>.
+     SMB1 is also still supported by <command>smbclient</command>.
+    </para>
+   </note>
+   <para>
+    In an existing setup, this can lead to issues in the following cases:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      If you are using an SMB server that does not support newer SMB protocol
+      versions. Windows has offered SMB 2.1 support since Windows 7 and
+      Windows Server
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      If you are reliant on SMB's Unix extensions which have not been ported
+      to newer protocol versions.
+     </para>
+    </listitem>
+   </itemizedlist>
+   <important>
+    <title>Following the Instruction Below Decreases System Security</title>
+    <para>
+     Following the instruction makes it possible to exploit security issues.
+     For more information, see
+     <link xlink:href="https://blogs.technet.microsoft.com/filecab/2016/09/16/stop-using-smb1/"/>.
+     Upgrade your server to allow for a more secure SMB version as soon as
+     possible.
+    </para>
+   </important>
+   <para>
+    If you run into either issue, you can enable support for SMB1 shares even
+    on the current &productname; kernel by adding the option
+    <option>vers=1.0</option> to the <command>mount</command> command line.
+   </para>
+  </sect2>
+  <sect2 xml:id="sec.samba.client.new-server">
+   <title>Enabling Newer Versions of the SMB Protocol on Clients</title>
+   <para>
+    To configure Samba to accept SMB 2.1, in the configuration file
+    <filename>/etc/samba/smb.conf</filename>, set the global parameter
+    <literal>server max protocol = SMB2_10</literal> (for a list of possible
+    values, see <command>man smb.conf</command>).
```

Comment:
'(for a list of possible  values, see <command>man smb.conf</command>)' -> make this a separate sentence without parenthesis